### PR TITLE
New Web Socket Order Updates for Smart API sss-75

### PR DESCRIPTION
**Pull request by Arvind-Acc**
New file created for Web Socket Order Updates for Smart API sss-75

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,15 +132,26 @@ token_list = [
         "tokens": ["26009"]
     }
 ]
-sws = SmartWebSocketV2(AUTH_TOKEN, API_KEY, CLIENT_CODE, FEED_TOKEN)
+# simple retry mechanism
+sws = SmartWebSocketV2(AUTH_TOKEN, API_KEY, CLIENT_CODE, FEED_TOKEN,max_retry_attempt=2, retry_strategy="simple", retry_delay=10, retry_duration=30)
```
Comment (anurag-amx) [inline README.md:136]: retry_strategy="simple" 
it should be 0 or in other int.

File: SmartApi/smartWebSocketOrderUpdate.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
+import ssl
+import websocket
+import time
+from logzero import logger
+
+class SmartWebSocketOrderUpdate(object):
+    ROOT_URI = "wss://tns.angelone.in/smart-order-update"
+    HEART_BEAT_MESSAGE = "ping"
+    HEART_BEAT_INTERVAL = 10
```
Comment (anurag-amx) [inline SmartApi/smartWebSocketOrderUpdate.py:9]: make variables correctly and provide comment

**Review by anurag-amx — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Arvind-Acc) [reply README.md]: Removed the simple and exponential from string comparison from websocket and use 0 and 1

File: SmartApi/smartWebSocketOrderUpdate.py (hunk 2 above)
Comment (Arvind-Acc) [reply SmartApi/smartWebSocketOrderUpdate.py]: make changes in variable name to give meaningfull name

**Review by anurag-amx — DISMISSED** [review]

**Review by gulshan178 — DISMISSED** [review]

**Pull request merged**